crates/bevy_pbr/src/atmosphere/mod.rs

```diff
@@ -124,52 +174,40 @@ impl Plugin for AtmospherePlugin {
             .contains(TextureUsages::STORAGE_BINDING)
         {
             warn!("AtmospherePlugin not loaded. GPU lacks support: TextureFormat::Rgba16Float does not support TextureUsages::STORAGE_BINDING.");
-            return;
+            break 'supported false;
         }
+        true
+    };
 
-        render_app
-            .init_resource::<AtmosphereBindGroupLayouts>()
-            .init_resource::<RenderSkyBindGroupLayouts>()
-            .init_resource::<AtmosphereSamplers>()
-            .init_resource::<AtmosphereLutPipelines>()
-            .init_resource::<AtmosphereTransforms>()
-            .init_resource::<SpecializedRenderPipelines<RenderSkyBindGroupLayouts>>()
-            .add_systems(
-                Render,
-                (
-                    configure_camera_depth_usages.in_set(RenderSystems::ManageViews),
-                    queue_render_sky_pipelines.in_set(RenderSystems::Queue),
-                    prepare_atmosphere_textures.in_set(RenderSystems::PrepareResources),
-                    prepare_atmosphere_transforms.in_set(RenderSystems::PrepareResources),
-                    prepare_atmosphere_bind_groups.in_set(RenderSystems::PrepareBindGroups),
-                ),
-            )
-            .add_render_graph_node::<ViewNodeRunner<AtmosphereLutsNode>>(
-                Core3d,
+    if supported {
+        commands.insert_resource(AtmosphereSupported);
+    }
```

Comment:
Instead of `break 'supported false` setting `supported` and then checking `supported`, I think it makes more sense to simply `return` and remove this `if supported` condition, it's functionally the same, but the early return is more recognizable as a pattern.